wot-ig-2023-draft.html

```diff
@@ -284,20 +284,25 @@ <h2>Scope</h2>
         <h3 id="plugfest">PlugFest and Testing</h3>
         <p>
           An important activity is the operation of PlugFests to collect interoperability implementation experience 
-          and to validate the current working assumptions of the potential building blocks discussed in the Web of Things Interest Group.
+          and to validate the current working assumptions of the potential building blocks and features discussed in the Web of Things Working Group task forces (e.g., Thing Description).
           These events enable developers to get together to test their implementations and facilitates networking between partners and experts.
           The Interest Group will seek to encourage work on open source projects and community evaluation of the Web of Things.
           In more detail, PlugFests allow participants to:
         </p>
         <ul>
-          <li>assist with preparing implementation reports for WoT WG Candidate Recommendations</li>
```

Comment:
this was deleted but really we should change "assist" to "provide test input to the WG for the purpose of preparing implementation reports"

Reply:
a new text below explain the role of the IG  and how it support to create test inputs